feat: help popover status button

Hallidayo opened the pull request:
## What kind of change does this PR introduce?

Studio Feature

## What is the current behavior?

On the Supabase Discord recently I've see seen some message to see if Supabase is down or issues with connectivity and links have been made to the [status](https://status.supabase.com/) website.

## What is the new behavior?

A button on the help overlay has been added so that users can just click on the button and see the status of Supabase easier.

Hallidayo committed:
status button added to help popover

Hallidayo committed:
ran prettier

MildTomato commented on studio/components/layouts/ProjectLayout/LayoutHeader/HelpPopover.tsx:
```diff
@@ -49,6 +50,13 @@ const HelpPopover: FC<Props> = () => {
+              <Link href="https://status.supabase.com/">
+                <a>
+                  <Button type="secondary" size="tiny" icon={<IconActivity />}>
+                    Status
```
maybe rename to "Supabase Status"

Hallidayo replied:
To get the buttons all aligned I've changed the 'Contact support team' to 'Contact support' and also the spacing has been made smaller:

<img width="395" alt="Screenshot 2022-08-08 at 11 24 42" src="https://user-images.githubusercontent.com/22655069/183397513-2cc4dace-f0f5-4a6a-b0e7-521f8f1667d5.png">

@MildTomato - Shall I push the commit?

saltcod replied:
Looks good @Hallidayo, give that a push!

Hallidayo replied:
@saltcod - Cool just pushed a commit.

Hallidayo committed:
button text changes

saltcod commented:
@Hallidayo A suggestion for the layout there, where it's a little tight: 

<img width="393" alt="CleanShot 2022-08-09 at 13 42 01@2x" src="https://user-images.githubusercontent.com/105593/183703454-1d66cf4f-7081-454c-9457-bce4eebc1764.png">


Also, the link html isn't quite right — you've got an extra <a> tag around the button. Should look like this I think:  

```
<Link passHref href={supportUrl}>
	<Button type="default" icon={<IconMail />} as="a">
		Contact Support
	</Button>
</Link>

```

Hallidayo committed:
removed un-needed anchor tags

Hallidayo committed:
button type prop changed to text

Hallidayo commented:
@saltcod - Odd I found that all three buttons were using un-needed anchor tags so I have removed them. I've also changed the button props `type` to `text`

saltcod commented:
Thanks @Hallidayo 

These should actually be links I think (a tags instead of buttons), since they are actual links. 

This should output an `a` tag, styled like the buttons that are there now.
```
<Link passHref href={supportUrl}>
	<Button type="default" icon={<IconMail />} as="a">
		Contact Support
	</Button>
</Link>
```

Hallidayo committed:
button as prop added

Hallidayo commented:
@saltcod - I've just added the `as` prop onto all three buttons.

saltcod commented:
Hey @Hallidayo so close!   You need the `passHref` in the `<Link>` as well: `<Link passHref href={supportUrl}>`

Hallidayo committed:
passHref prop added to Links

Hallidayo commented:
@saltcod - I think we might be their now sorry :laughing:

saltcod commented:
@Hallidayo no worries at all! Thanks for sticking with it!  :^)
